webapp/src/main/resources/public/js/sdk/entity/BranchStatisticsContent.js

```diff
@@ -39,7 +45,7 @@ export default class BranchStatisticsContent {
          *
          * @type {boolean}
          */
-        this.isTruncated = false;
+        this.paginated = false;
```

Comment:
Nit: I prefer the scheme of `isX` for booleans. So I think `isPaginated` would be a better name

Reply:
Done